src/webview/webview2/mod.rs

```diff
@@ -131,11 +144,24 @@ impl InnerWebView {
             // remove "mini menu" - See https://github.com/tauri-apps/wry/issues/535
             // and "smart screen" - See https://github.com/tauri-apps/tauri/issues/1345
             format!(
-              "--disable-features=msWebOOUI,msPdfOOUI,msSmartScreenProtection{}",
+              "--disable-features=msWebOOUI,msPdfOOUI,msSmartScreenProtection{}{}",
               if autoplay {
                 " --autoplay-policy=no-user-gesture-required"
               } else {
                 ""
+              },
+              if let Some(proxy_setting) = proxy_config {
+                match proxy_setting.proxy_connection {
+                  ProxyConnection::Http(endpoint) => {
+                    format!(" --proxy-server=http://{}:{}", endpoint.host, endpoint.port)
+                  }
+                  ProxyConnection::Socks5(endpoint) => format!(
+                    " --proxy-server=socks5://{}:{}",
```

Comment:
we should update the documentation of `WebViewBuilderExtWindows::with_additional_browser_args` to mention this.